src/dmd/declaration.d

```diff
@@ -1088,6 +1088,9 @@ extern (C++) class VarDeclaration : Declaration
     VarDeclaration lastVar;         // Linked list of variables for goto-skips-init detection
     uint endlinnum;                 // line number of end of scope that this var lives in
 
+    /// it was declared in an anon union
+    bool isAnonUnionMember;
```

Comment:
When changing the layout of `extern (C++)` aggregates, be use to update the header files as well.

Reply:
yes fixup coming

Reply:
https://github.com/dlang/dmd/pull/9290